content/en/platform/corda/4.10/enterprise/tools-config-obfuscator.md

```diff
@@ -21,6 +21,10 @@ The Configuration Obfuscator makes node installation less vulnerable to someone
 Although the Configuration Obfuscator does protect the node against trawling attacks, it does not ensure that password protection is completely secure. For improved security, always use the latest released version of the tool.
 {{< /warning >}}
 
+{{< note >}}
+Obfuscation is supported exclusively for Corda-related configuration files, such as node or other core components. It is not available for CorDapps or non-Corda-specific configuration files.
```

Comment:
Reading this next to the note that follows about using it with CENM makes me think this could be construed as contradictory, so we should combine the two.

Reply:
Combined.